### DISCUSSION TITLE
Is there any documentation for what buttonpin does?

### DISCUSSION THREAD

**Discussion by electromechpro** [ldijkman/randomnerd_esp32_wifi_manager#31] [Q&A]
I was looking in the code for what happens when button pin is switched, hoping for a manual trigger of the relay.  I was unable to see anything there.  Is there support for the button pin yet?  Also, is there an option to turn on the relay and have it stay on until turned off?
I have Electra woeking on a Wemos D1 mini clone with relay shield.

**Comment by ldijkman** [comment]
Maybe have a look at

https://ldijkman.github.io/async-esp-fs-webserver/

Similar but switches the LED onboard
